fetch.bs

```diff
@@ -8713,83 +8713,141 @@ relevant to you. The rest of the parameters are used less frequently, often for
 and they are documented in detail in the [[#requests]] section of this standard.
 
 
-<h3 id=fetch-elsewhere-fetch class=no-num>Invoking fetch</h3>
+<h3 id=fetch-elsewhere-fetch class=no-num>Processing Responses</h3>
 
-<p>Aside from a <a for=/>request</a> the <a for=/>fetch</a> operation takes several optional
-arguments. For those arguments that take an algorithm: the algorithm will be called from a task (or
-in a <a for=/>parallel queue</a> if <a for=fetch><i>useParallelQueue</i></a> is true).
+<p>Once the <a for=/>request</a> is set up, determine how you would like to process the
+<a for=/>response</a>, and in particular at what stage you would like to receive a callback:
```

Comment:
This now lacks the introduction that you need to invoke fetch. I think we should keep that as the text below refers to it.

Reply:
I don't exactly understand this comment.

Reply:
@annevk which introduction is missing? The previous "invoking fetch" section was mainly a description of the different parameters. This PR still describes all the parameters, but from the point of view of "what is the caller trying to do". Which part is missing in your opinion?

Reply:
What I mean is that it doesn't mention the fetch operation at all in this introductory paragraph.

Reply:
I've reworded a bit. Does this look better?

Reply:
Yeah, that seems a lot better, thanks.